src/javascripts/modules/index.js

```diff
@@ -0,0 +1,31 @@
+/*
+  Automatically instantiates modules based on data-attributes
+  specifying module file-names.
+*/
+
+const moduleElements = document.querySelectorAll('[data-module]')
+
+for (var i = 0; i < moduleElements.length; i++) {
+  const el = moduleElements[i]
+  const name = el.getAttribute('data-module')
+  const Module = require(`./${name}`).default
+  new Module(el)
```

Comment:
This is another default blendid file, but it might be useful once you get to the stage where you're adding JS stuff. This bit of code will load a corresponding JS file that has the same name as the value of the `data-module` attribute, so `data-module="foo"` would load and initialize **foo.js**.